### PR TITLE
Add drive name to status message

### PR DESCRIPTION
**Pull request by bictorv**

### CODE REVIEW

**Comment by bictorv** [comment]
So it's only the latest commit. Sorry, should have created a new branch I suppose.

**Comment by larsbrinkhoff** [comment]
Try this: `git fetch` and then `git rebase origin/master`.  If the result looks good, `git push -f`

**Comment by bictorv** [comment]
Thanks, seems to have worked!

File: rtape.c [hunk 1]
```diff
@@ -408,6 +413,12 @@ static void send_status(int id, const char *message)
   buf[2] = (id >> 8) & 0xFF;
   buf[33] = flags & 0xFF;
   buf[34] = (flags >> 8) & 0xFF;
+  memset(&buf[1+2+3+3+3+1+2+2], 0, 16);
```
Comment (larsbrinkhoff) [inline rtape.c:416]: It would be a good idea to have a symbolic constant for this.  Maybe later...

**Comment by larsbrinkhoff** [comment]
Thanks!  Guessing some Lispm want the drive name in the status message?

**Comment by bictorv** [comment]
Symbolic constants would be good. Drive names are useful, yes, not absolutely crucial but useful.

**Pull request merged**